carceral-groups-website/.env

```diff
@@ -1 +1,3 @@
-VITE_CLIENT_ID=5f7f903f-d4a6-476d-9894-807740a596df
\ No newline at end of file
+VITE_CLIENT_ID=5f7f903f-d4a6-476d-9894-807740a596df
+VITE_AZURE_BLOB_SAS_URL=
```

Comment:
Still need to figure out how to add these details in during build and deployed to Azure Storage in order to not share secrets.